script/Ciel/Day 7/07_01_CIEL07_2_1A.txt

```diff
@@ -27,21 +27,22 @@ I get the feeling she's still condemning me for my conversation with Noel this m
 [sha:2eba0d00a2103e4911ea29b2e0f36e64d407dafd]{
 -- Page 174, Offset 15217.
 -- 「そ、そんなコトは訊いてないし気にしてませんけど。
-"T-that wasn't what I was asking about, and I wasn't worried about it either.
+"T-That isn't what I was asking about, and it's not like I'd care if you did.
 }
 [sha:0c32043756a9140c3c19841305cdf3b6cd46835f]{
 -- Page 174, Offset 15218.
 -- ……そうですか。あれはノエル先生の、いつもの暴走片道切符なんですね。まあそんなコトはどうでもいいので早く座ってください。立っていられると落ち着かないです」
 // 暴走片道切符 is so weird here, I don't even know where to begin. It sort of makes sense that its signifying something about Noel having poor impulse control (especially considering how crazy we see her get later in the route), but the phrasing is so odd I don't really know what to translate it as.
 // 落ち着かない here (translated as 'bothered' is reflected a few lines down at Offset 15226.
-But I see... So that was just part of Noel-sensei's usual one-way trip to going out of control.
+But I see...
+ I guess that was just typical Noel-sensei behavior――she's a lost cause the minute she flies off the handle.
  Well, who really cares. Please take a seat. It bothers me having you stand around like that."
 }
 [sha:5d90093217eec81aaa065ade68775341bd8951a3]{
 -- Page 175, Offset 15219.
 -- 先輩に促されて畳に正座する。
 // similar to Offset 15245 but no seiza there...
-Prompted by Senpai, I take a straight seated posture on the tatami floor.
+Prompted by Senpai, I sit at attention on the tatami floor.
```

Comment:
Interesting interpretation of seiza

Not sure if I quite agree

Reply:
'straight seated posture' just feels so bit wooden / literal. It kinda screams "I am translating seiza" in a way I'd prefer to avoid. At attention carries the same sort of uprightness imo.